DRAFT.md

```diff
@@ -726,45 +726,40 @@ explicit constexpr indirect(allocator_arg_t, const Allocator& a, U&& u, Us&& ...
 constexpr indirect(const indirect& other);
 ```
 
-18. _Mandates_: `T` is a complete type.
```

Comment:
Why is this no longer true?  I would expect `T` must be complete so we can call the underlying copy constructor

Reply:
My reasoning was that we're not imposing any constraints on `T`. The copy could be done in another TU and linked in.